src/_includes/steering-faq.html

```diff
@@ -0,0 +1,103 @@
+
+  <div class="accordion" id="steering-faq">
+
+    <div class="card">
+
+      <div class="card-header" id="headingOne">
+        <h3 class="mb-0">
+          <button class="btn btn-link btn-block text-left collapsed" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
+            About the meeting
+          </button>
+        </h3>
+      </div>
+
+      <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#steering-faq">
```

Comment:
I'm not sure if you need the `aria-labelledby` because the card itself gets skipped by the screen reader because it doesn't have a `role=""` (I did a quick test in VoiceOver to confirm, but there's always a bit of variation depending on OS + browser + screen reader). In VoiceOver at least it gets skipped.

It's probably not worth giving the card a `role` as it wouldn't aid the efficiency of the reading experience (it would add an additional stop but I think the headings confer enough context already).

Reply:
That's great to know, thank you. I'm still wrapping my head around roles and such and need to spend more time with screen readers when testing. This was mainly a copy-paste job from the bootstrap components.

Reply:
> I'm still wrapping my head around roles and such

Yeah it's very confusing. My understanding is that most elements which are not semantically significant (div, span, etc) will either get pruned out of the accessibility tree, or just ignored by the screen reader cursor. If one of those elements has a role applied then it can make it semantically significant. If I recall, in this case I think the card class is on a div without a role, so in my testing it got bypassed by the screen reader.

Reply:
I may leave it until I play with it a bit more.